packages/sources/ncfx-test/src/config/index.ts

```diff
@@ -0,0 +1,31 @@
+export const CRYPTO_DEFAULT_BASE_WS_URL = 'wss://feed.newchangefx.com/cryptodata'
+export const FOREX_DEFAULT_BASE_WS_URL =
+  'wss://fiat-ws.eu-west-2.apingxelb.v1.newchangefx.com/sub/fiat/ws/ref'
+
+// Needed for README generator
+export const DEFAULT_ENDPOINT = 'crypto'
```

Comment:
We shouldn't, the README generator should take this from it's proper place in adapter.defaultEndpoint

Reply:
Roger that